spec/requests/api/v1/documents_spec.rb

```diff
@@ -287,61 +287,71 @@
-                          language: 'en',
-                          title: 'hi there 4',
-                          description: 'bigger desc 4',
+        create_document({ audience: 'Everyone',
                           content: 'huge content 4',
                           created: 2.hours.ago,
-                          updated: Time.zone.now,
+                          description: 'bigger desc 4',
+                          language: 'en',
+                          path: 'http://www.gov.gov/url4.html',
                           promote: true,
-                          path: 'http://www.gov.gov/url4.html' }, document_repository)
+                          searchgov_custom2: 'comma, separated, custom, content',
+                          searchgov_custom3: 123,
+                          title: 'hi there 4',
+                          updated: Time.zone.now,
+                          id: id },
+                        document_repository)
 
         put_document
       end
 
       it 'returns success message as JSON' do
-        expect(response.status).to eq(200)
-        expect(JSON.parse(response.body)).
+        expect(response).to have_http_status(:ok)
+        expect(response.parsed_body).
           to match(hash_including('status' => 200,
                                   'developer_message' => 'OK',
                                   'user_message' => 'Your document was successfully updated.'))
       end
 
       it 'updates the document' do
         document = document_repository.find(id)
-        expect(document.path).to eq('http://www.next.gov/updated.html')
-        expect(document.promote).to be_falsey
-        expect(document.title).to eq('new title')
-        expect(document.description).to eq('new desc')
-        expect(document.thumbnail_url).to eq('https://18f.gsa.gov/assets/img/logos/new/18F-Logo-M.png')
-        expect(document.content).to eq('new content')
-        expect(document.content_type).to eq('website')
-        expect(document.tags).to match_array(['new category'])
         expect(document.changed).to eq('2016-01-01T10:00:01Z')
         expect(document.click_count).to eq(1000)
+        expect(document.content_type).to eq('website')
+        expect(document.content).to eq('new content')
+        expect(document.description).to eq('new desc')
         expect(document.mime_type).to eq('text/plain')
+        expect(document.path).to eq('http://www.next.gov/updated.html')
+        expect(document.promote).to be_falsey
+        expect(document.searchgov_custom1).to contain_exactly('custom content with spaces')
+        expect(document.searchgov_custom2).to contain_exactly('new', 'comma', 'separated', 'custom', 'content')
+        expect(document.tags).to contain_exactly('new category')
+        expect(document.thumbnail_url).to eq('https://18f.gsa.gov/assets/img/logos/new/18F-Logo-M.png')
+        expect(document.title).to eq('new title')
+      end
+
+      it 'persists unchanged attributes' do
```

Comment:
This is the only meaningful change that wasn't simply rubocop cleanup/alphabetization of attributes.